core-js/src/main/javascript/cdf/components/MultiButtonComponent.js

```diff
@@ -143,8 +143,14 @@ define([
           if(!myself.isMultiple) { break; }
         }
       }
+	  var useFirstValue = true;
```

Comment:
_useFirstValue_ and _defaultIfEmpty_ serve the same purpose, we only need one variable to control the behavior of selecting the first value if none is selected. The same applies for the other components.